app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,88 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.Exception.TaskNotFoundException;
+import com.codesoom.assignment.TaskManager;
+import com.codesoom.assignment.models.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+/**
+ * Task 처리에 대한 Request를 받고, 이에 대한 Response를 반환하는 작업을 담당합니다.
+ *
+ * @see RestController
+ * @see RequestMapping
+ * @see CrossOrigin
+ */
```

Comment:
클래스 주석을 작성할 때에는 클래스의 책임에 집중합시다. 책임이 가장 중요합니다. 컨트롤러는 무엇을 책임지고 있을까요? 바로 HTTP 프로토콜입니다. 컨트롤러가 HTTP 프로토콜 처리를 책임지고 있기 때문에 HTTP 프로토콜의 용어와 기능은 컨트롤러까지만 도달합니다. 그래서 컨트롤러 뒤에 있는 서비스(`TaskManager`)는 HTTP를 전혀 알지 못해도 잘 돌아가는 것입니다. 서비스는 요청이 HTTP를 통한 것인지 아니면 운영체제 인터럽트에 의한 것인지, 마우스 클릭에 의한 것인지, 쉘 명령에 의한 것인지 몰라도 됩니다. 컨트롤러가 앞에서 자기 책임을 다하며 막아주고 있기 때문이죠.

```suggestion
/**
 * HTTP 프로토콜을 통한 Task 처리 컨트롤을 담당합니다.
 */
```

아래쪽에 있는 `@see` 시리즈는 불필요합니다. `@see`는 "이 클래스의 도메인 로직을 이해하려면 이것은 반드시 봐야 합니다."의 의미로 사용해 보세요.

Reply:
@see의 의미에 그런 의미를 담고 있는 지 몰랐네요. 저는 해당 클래스에서 사용하고 있는 게 있으면 무조건 사용하는 건지 알았습니다.

Reply:
이렇게 @see 라고 작성하면 see 라는 아이디를 가진 다른 github 유저에게 알림이 가게 됩니다. 여기까지 와서 읽어볼 수도 있겠죠. 그래서 이런 경우엔 양쪽에 ``` ` ``` 을 씌워주도록 해보세요. `@see` 이렇게 하면 링크가 안 걸립니다.
```
`@see`
```

Reply:
다른 사람을 배려하지 않고 사용했네요. ㅎㅎ
감사합니다. ㅎㅎ